cmd/test-server/handlers.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"encoding/json"
+	"net"
+	"net/http"
+	"path"
+	"text/template"
```

Comment:
This should be "html/template", or the client side JS errors out. 
```suggestion
	"html/template"
```

Reply:
Whoops. Fixed in 3a99757339f351ea3bfc0ec535172298b2b9b16a.